backend/app/domains/habits/schemas.py

```
@@ -0,0 +1,78 @@
+    description: str | None = Field(default=None, max_length=255)
+    category: str | None = Field(default=None, max_length=100)
+    schedule: Dict[str, Any] = Field(default={"type": "daily"})
+    start_date: date | None = Field(default=None)
+    backfill_window_hours: int = Field(default=48, ge=0, le=168)  # Max 1 week
+    
+    @validator('schedule')
+    def validate_schedule(cls, v):
+        """Validate schedule format."""
+        if not isinstance(v, dict):
+            raise ValueError('Schedule must be a dictionary')
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)